local-development.md

```diff
@@ -0,0 +1,35 @@
+# Testing locally
```

Comment:
Do we want to add this file as an exclude in the _config.yml so it does not general a top level page on the site itself? Seems more like a GitHub thing.

Reply:
Yeah - I missed that on my testing, that it rendered as a page on the final site. :/ Doing one more PR.

Reply:
Too late! I already merged it. :-)